README.md

```diff
@@ -165,6 +165,36 @@ MarkLogic Server Docker containers are configured using a set of environment var
 | LICENSE_KEY           | license key                     | no                                | n/a       | set MarkLogic license key                          |
 | LICENSEE            | licensee information            | no                                | n/a       | set MarkLogic licensee information                 |
 |INSTALL_CONVERTERS   | true                            | no                                | false     | Installs converters for the client if they are not already installed | 
+|OVERWRITE_ML_CONF   | true                            | no                                | false     | Deletes and rewrites `/etc/marklogic.conf` with the passed in env variables if set to true | 
+
+
+MarkLogic Server also can be configured through a configuration file on the image to `/etc/marklogic.conf`. To interface with the configuration file pass in `OVERWRITE_ML_CONF` set to `true` and the following env variables can be written, if set, to `/etc/marklogic.conf`. 
```

Comment:
"MarkLogic Server also can be configured through a configuration file on the image at...

"To change the configuration file, pass in the parameter.... set to 'true. The following env variables can also be written to the '/etc/marklogic.conf' if the parameter is set.

Reply:
09912a67e64860f79171c63e9bc7e7e432f8d02b Updated